components/event-publisher-proxy/Makefile

```diff
@@ -23,5 +23,8 @@ clean: resolve_clean
 resolve_clean:
 	rm -rf vendor
 
-build-local: test-local 
+build-local: test-local
 
+.PHONY: generate
+generate:
```

Comment:
would it make sense to have the generate target as a dependency on other targets such as `test`?

Reply:
we agreed that this will part of a bigger ticket for recreating a makefile without dependency on `generic-make-go.mk`